mne/preprocessing/maxwell.py

```diff
@@ -341,29 +343,32 @@ def maxwell_filter(raw, origin='auto', int_order=8, ext_order=3,
     # Reconstruct raw file object with spatiotemporal processed data
     max_st = dict()
     if st_duration is not None:
-        max_st.update(job=10, subspcorr=st_correlation, buflen=st_duration)
+        max_st.update(job=10, subspcorr=st_correlation,
+                      buflen=st_duration / info['sfreq'])
         logger.info('    Processing data using tSSS with st_duration=%s'
-                    % st_duration)
+                    % max_st['buflen'])
         st_when = 'before' if st_fixed else 'after'  # relative to movecomp
     else:
-        st_duration = min(raw_sss.times[-1], 10.)  # chunk size
+        st_duration = raw_sss.time_as_index(10., use_rounding=True)  # chunksz
```

Comment:
Maybe just put this on the line above if you want the comment. Staying under 80 char is now making this unclear :p